pubsublite/streaming-analytics/README.md

```diff
@@ -0,0 +1,107 @@
+# Pub/Sub Lite with Cloud Dataflow
+
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=pubsublite/streaming-analytics/README.md)
+
+Samples showing how to use [Pub/Sub Lite] with [Cloud Dataflow].
+
+## Pub/Sub Lite to Cloud Storage sample
+
+[PubsubliteToGcs.java](examples/PubsubliteToGcs.java)
+
+This sample shows how to create an [Apache Beam] streaming pipeline that reads
+messages from [Pub/Sub Lite], group the messages using a fixed-sized windowing
+function, and writes them to [Cloud Storage].
+
+Resources needed for this example:
+
+1. A pair of Pub/Sub Lite topic and subscription. 
+2. A Cloud Storage bucket.
+
+### Setting up
+
+1. [Enable the APIs](https://console.cloud.google.com/flows/enableapi?apiid=dataflow,compute_component,logging,storage_api,pubsublite.googleapis.com): Cloud Dataflow, Compute Engine, Cloud Logging, Cloud Storage, Pub/Sub Lite.
```

Comment:
To be very explicit, we should at least mention you need a Google Cloud project with billing enabled, and the `GOOGLE_APPLICATION_CREDENTIALS` variable points to a service account.

Otherwise, the Dataflow job will fail, even in the DirectRunner since it needs to access GCP resources.

Reply:
I added some setup instructions with specific permissions to give. Also just tested it myself.